expfmt/text_create.go

```diff
@@ -330,32 +324,62 @@ func writeSample(
 	return written, nil
 }
 
-// writeLabelPairs converts a slice of LabelPair proto messages plus the
+// writeNameAndLabelPairs converts a slice of LabelPair proto messages plus the
 // explicitly given additional label pair into text formatted as required by the
 // text format and writes it to 'w'. An empty slice in combination with an empty
 // string 'additionalLabelName' results in nothing being written. Otherwise, the
 // label pairs are written, escaped as required by the text format, and enclosed
 // in '{...}'. The function returns the number of bytes written and any error
 // encountered.
```

Comment:
The doc comment here would benefit from an update for the new features (additionally passed `name`, special syntax/quoting for `name` and label names containing formerly invalid characters).

Reply:
done